csrankings-y.csv

```diff
@@ -514,7 +514,7 @@ Yongtao Wang,Peking University,http://www.icst.pku.edu.cn/cpdp/index.php/researc
 Yongwang Zhao,Zhejiang University,https://person.zju.edu.cn/zhaoyw,NOSCHOLARPAGE
 Yongwei Wu,Tsinghua University,http://madsys.cs.tsinghua.edu.cn/~yongweiwu,ld3axtkAAAAJ
 Yongxin Tong,Beihang University,http://sites.nlsde.buaa.edu.cn/~yxtong,aeCHfDIAAAAJ
-Yongxin Yang,University of Surrey,https://yang.ac,F7PtrL8AAAAJ
+Yongxin Yang,Queen Mary University of London,https://yang.ac,F7PtrL8AAAAJ

```

Comment:
Not in CS, please clarify.

Reply:
I’m in the Department of Computer Science. Here is my profile: http://eecs.qmul.ac.uk/people/profiles/-yongxin-yang.html At QMUL, Department of Computer Science is part of School of Electrical Engineering and Computer Science (EECS), I’m in CS branch, not EE.